### ISSUE TITLE
[Serious] Placeholder text lacks 4.5:1 contrast ratio

### ISSUE THREAD

**Issue by ntallman** [APTrust/registry#62]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87642 |
| **Checkpoint** | 1.4 Distinguishable › Color Contrast (regular text) (1.4.3.a) |
| **Impact** | Serious |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 02 Account / Change Password |
| **Test Case Name** | AP Trust Registry (Demo) |
| **URL** | <https://demo.aptrust.org/users/my_account> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Product** | Website |
| **Date Reported** | 05/08/26 at 11:26 am |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | a3caccess+@gmail.com |

## Description

The contrast ratio between placeholder text and its background is not at least 4.5:1.
Global: the text edit fields have very faint placeholder text that fails minimum color contrast requirements. Since this duplicates the visible label, it is not necessary. See Work Items User Experience Video at 16:20.

## Recommendation

Rule : 
Small text (under 18 point regular font or 14 point bold font) MUST have a contrast ratio of at least 4.5 to 1 with the background. This issue applies to placeholder text as well.

How To Fix: 
Fix this issue by adjusting the color of the placeholder text to increase the contrast to at least 4.5 to 1.

Reference: 
W3C-WAI tutorial: https://www.w3.org/WAI/tutorials/forms/instructions/#styling
The Problem with Placeholders: https://adamsilver.io/blog/the-problem-with-placeholders-and-what-to-do-instead/

Background: 
People who have low vision or are colorblind may have difficulty reading text if the contrast between the text its background is insufficient. When the contrast ratio between text and its background is adequate, people who have low vision or are colorblind are more likely to be able to read the text.

## Source Code

```
N/A
```

## Screenshots

![04_placeholder text](https://raw.githubusercontent.com/APTrust/registry/master/docs/accessibility-audit/screenshots/87642/04_placeholder_text.png)
![02_color contrast placeholder](https://raw.githubusercontent.com/APTrust/registry/master/docs/accessibility-audit/screenshots/87642/02_color_contrast_placeholder.png)

**Comment by mjiori** [comment]
Ablr verified this fix in demo: Marked as "Fixed, RJ, 8/17/26"

The fix is now in prod. Closing.
